web/ts/HelloWorld.ts

```diff
@@ -0,0 +1,6 @@
+// This TypeSCript (.ts) file is part of JMRI.  If you see this comment in a JavaScript (.js)
```

Comment:
See the letter C in TypeS**C**ript above.

I wouldn't normally complain about this, but if this file is the template for future TypeScript files, it would be good to get it right from the beginning.